CHANGES.rst

```diff
@@ -20,6 +122,65 @@ Fixes:
 
 - Fixed displaying the body text of a feed item.  This is when
   ``render_body`` is switched on in the Syndication settings.
+- Only encode JS body if unicode in gruntfile generation script to avoid
+  unicode error.
+  [jensens]
+
+- Only encode CSS body if unicode in gruntfile generation script to avoid
+  unicode error.
+  [rnix]
+
+- Gruntfile failed if only css or only javascripts were registered.
+  [jensens]
+
+- Bundle aggregation must use ++plone++static overrided versions if any.
+  [ebrehault]
+
+- Fix bundle aggregation when bundle has no CSS (or no JS)
+  [ebrehault]
+
+- Fix relative url in CSS in bundle aggregation
+  [ebrehault]
+
+
+5.1a1 (2016-03-31)
```

Comment:
@ebrehault you're including part of the changelog from 5.1 and to me that makes no sense.